railties/lib/rails/app_updater.rb

```diff
@@ -21,14 +21,14 @@ def app_generator
       private
         def generator_options
           options = { api: !!Rails.application.config.api_only, update: true }
-          options[:skip_javascript] = !File.exist?(Rails.root.join("bin", "yarn"))
           options[:skip_active_record]  = !defined?(ActiveRecord::Railtie)
           options[:skip_active_storage] = !defined?(ActiveStorage::Engine) || !defined?(ActiveRecord::Railtie)
           options[:skip_action_mailer]  = !defined?(ActionMailer::Railtie)
           options[:skip_action_cable]   = !defined?(ActionCable::Engine)
           options[:skip_sprockets]      = !defined?(Sprockets::Railtie)
           options[:skip_bootsnap]       = !defined?(Bootsnap)
           options[:skip_spring]         = !defined?(Spring)
+          options[:webpack]             = File.exist?(Rails.root.join("bin", "yarn"))
```

Comment:
```suggestion
          options[:webpack]             = File.exist?(Rails.root.join("config", "webpacker.yml"))
```

not sure how common this is but if someone's using npm instead of yarn this wouldn't cover them.

Reply:
ps. very excited for this PR. 👋  webpack